ambry-store/src/test/java/com.github.ambry.store/DiskSpaceAllocatorTest.java

```diff
@@ -358,6 +342,35 @@ private void runConcurrencyTest(List<DiskSpaceRequirements> requirementsList, fi
   }
 
   /**
+   * Assert that an initialization error occurs when permissions on a reserve directory are modified.
+   * @param directoryToRestrict the directory to make unreadable or unwritable.
+   * @param restrictReadOrWrite {@code true} to make the directory unreadable, or {@code false} to make it unwritable
+   * @param requirements the {@link DiskSpaceRequirements} to provide to
+   *                     {@link DiskSpaceAllocator#initializePool(Collection)}
+   */
+  private void runInitFailureTest(File directoryToRestrict, boolean restrictReadOrWrite,
```

Comment:
shouldn't you name this variable "restrictRead"

Reply:
addressed